### ISSUE TITLE
Unable to configure properly SublimeANSI for 'tput reset' output

### ISSUE THREAD

**Issue by maxiimou** [aziz/SublimeANSI#20]
Hello !
On my machine ``tput reset`` ouputs the string ``\033(B\033[m``.

I've been trying to change the configuration in the ansi.sublime-settings so that the ANSI_FG white scope match it but it doesn't seems to be working... here what I tryed:

```
      {"scope": "white", "code": "(\\x1b\\[[(]B)?\\x1b\\[(0;)?(37)?m", "color": "#c7c7c7"},
      {"scope": "white", "code": "(\\x1b\\[\\(B)?\\x1b\\[(0;)?(37)?m", "color": "#c7c7c7"},
```

I pre-pened the original string (which should match the ``\033[m``) and added ``(\\x1b\\[[(]B)?`` or ``(\\x1b\\[\\(B)?`` to detect the leading part.

This technique is no effective, do you have any idea ?
Thanks a lot!

**Comment by jakkubu** [comment]
Can you send the whole output string?

**Comment by maxiimou** [comment]
``tput sgr0 > reset_str`` outputs (visualized using vim or less): ``^[(B^[[m``. 

I suppose ``^[`` stand for the ``ESC`` character (decimal 27, hex 0x1B, octal 033) so the string would be (char separated by commas) ``ESC, (, B, ESC, [, m``. One of the tool I use ouputs each lines with this sequence.


According to [this site](http://ascii-table.com/ansi-escape-sequences-vt-100.php) the ``ESC ( B`` sequence seems to be used for "Set United States G0 character set". So I ended up forking SublimeANSI and tried to add this code as unsupported (see [this commit](https://github.com/maxiimou/SublimeANSI/commit/09c06c8348de11e93df2b4d5df00fb40dca0cc3d)). 
First it seemed to work, but finally I may not have installed the ANSIescape package properly from my github repo and I can not test anymore...

**Comment by jakkubu** [comment]
Currently this plug-in does not support codes for VT100 terminal. It deletes only the commands from [this](http://ascii-table.com/ansi-escape-sequences.php) list.

I plan to rewrite some part of this plugin in near future (hope to find some time). I think all regexps should be set up in ANSI settings, so everybody would be able to add any other ANSI character without changing the code.

The lines you edited could be removed since they are probably not necessary (need to test it). Your edits works, because it changes the ANSI VT100 code to the one that will be deleted later.

Adding the codes for deletion should be done in [line 164](https://github.com/aziz/SublimeANSI/blob/3e49dc6531177e0fcdf27c9e5dd225d8d86865ed/ansi.py#L164) (for `on_finish` trigger) and in [line 325](https://github.com/aziz/SublimeANSI/blob/3e49dc6531177e0fcdf27c9e5dd225d8d86865ed/ansi.py#L325) (`on_data` trigger).

**Comment by maxiimou** [comment]
Yes! It would be great that settings allow users to define ignored/unsupported codes via regex.

I may have the time to submit a pull request in this way tonight if I find the way to test my development on my computer. 

Is it supposed to work (the ANSIescape package) directly once copied on the Package directory ? The installation through "package control" worked for me but this way no...
